Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -109,8 +109,12 @@ void Processor::initialize(u32)
         asm volatile("wfi");
 }
 
-void Processor::flush_tlb_local(VirtualAddress, size_t)
+void Processor::flush_tlb_local(VirtualAddress vaddr, size_t page_count)
 {
+    // NOTE: We must flush the data cache here, such that the updated page tables actually
+    //       end up in main memory, and thus actually get used by the MMU.
+    Aarch64::Asm::flush_data_cache(vaddr.get(), 4096 * page_count);
```

Comment:
`s/4096/PAGE_SIZE` (or is that no available here?)

Reply:
Should be available in all Serenity-only code; on Lagom it sometimes expands to a `sysconf` call.

Reply:
Yep, good catch, forgot to use that.